.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
The plan is, like you said, to :
- remove the line, and commit this removal
- run `git rebase -i master` to merge these 2 commits, keeping the commit message of the first one (as it did not mention .gitignore)
- check with `git log` that you have only one commit remaining
- push to github with `--force` option, which will update the PR